Cpp/fostgres-fg/fg.builtin.http.cpp

```diff
@@ -85,6 +85,12 @@ namespace {
                 fg::json::const_iterator end) {
         return nobody(&fg::testserver::get, stack, pos, end);
     }
+    fg::json
+            head(fg::frame &stack,
+                fg::json::const_iterator pos,
+                fg::json::const_iterator end) {
+        return nobody(&fg::testserver::head, stack, pos, end);
```

Comment:
I expect this should return `{}` instead as there is never any body in HEAD. It should also not take all of the arguments that `nobody` uses because there is never a body returned.

Reply:
I don't get this part "t should also not take all of the arguments that nobody uses", but aren't the parameters (stack, pos, end) required to get the view name, path, and status to check?

Reply:
Yes, but the very last argument is the object to check the body against. Now it is optional so technically it's already correct. I just think it'd be clearer if having that final argument causes an error saying that there will be no return body to check against

Reply:
This is probably wrong isn't it. Probably none of the functions check that they've exhausted the passed arguments -- they just ignore the extras so no need to do anything special here yet either.

I guess a future improvement should be to add a warning and later on an error for extra arguments.